source/new-ia/by-role/developer/rest-services/get-a-list-of-sites.rst

```diff
@@ -0,0 +1,46 @@
+:is-up-to-date: True
+:last-updated: 4.0.3
+
+:nosearch:
+
+================================================
+Get the Names of Sites Running in Crafter Engine
```

Comment:
I'm not sure where this should go, if anywhere, it's an example more than anything. I can see why it was in a "cookbook" or similar, but not sure where it would go in the new IA. Worth a conversation.